account-kit/smart-contracts/src/ma-v2/modules/paymaster-guard-module/module.ts

```diff
@@ -0,0 +1,49 @@
+import { encodeAbiParameters, type Address, type Hex } from "viem";
+import { paymasterGuardModuleAbi } from "./abis/paymasterGuardModuleAbi.js";
+
+const addresses = {
+  default: "0x00", //dummy address, replace with deployed module address?
```

Comment:
If you want the zero address, you can import `zeroAddress` from viem. I'm not sure that the viem APIs that accept addresses will work with `"0x00"`.

Reply:
`addresses` is no longer being used in the module definition, resolving this comment!